packages/core/src/ceramic.ts

```diff
@@ -244,7 +244,10 @@ export class Ceramic implements StreamReaderWriter, StreamStateLoader {
 
     this._ipfs = modules.ipfs
 
-    this._schemaValidator = new SchemaValidation()
+    const numCores = os.cpus().length
+    // Use number of threads equal to half the available cores for schema validation. Leave the
+    // other half for signature validation.
+    this._schemaValidator = new SchemaValidation(Math.floor(numCores / 2))
```

Comment:
should this be at least 1? maybe `Math.max(1, Math.floor(numCores / 2)`?

Reply:
also, looks like we need to call `this._schemaValidator.init()` still

Reply:
Ha, I know that feeling 😅 Definitely did have to call `init` in a bunch of tests as well. LGTM.